langchain/src/experimental/openai_assistant/index.ts

```diff
@@ -76,15 +82,18 @@ export class OpenAIAssistantRunnable<
       model,
     });
 
-    return new this<RunInput, RunOutput>({
+    return new this({
```

Comment:
Shouldn't you pass `AsAgent` through here?

Reply:
It's inferred from the value of `asAgent`